system/sensord/rawgps/rawgpsd.py

```diff
@@ -120,13 +122,37 @@ def setup_quectel(diag: ModemDiag):
 
   if gps_enabled():
     at_cmd("AT+QGPSEND")
+  #at_cmd("AT+QGPSDEL=0")
 
   # disable DPO power savings for more accuracy
   at_cmd("AT+QGPSCFG=\"dpoenable\",0")
   # don't automatically turn on GNSS on powerup
   at_cmd("AT+QGPSCFG=\"autogps\",0")
 
-  at_cmd("AT+QGPSSUPLURL=\"supl.google.com:7275\"")
+  # Do internet assistance
+  at_cmd("AT+QGPSXTRA=1")
+  assist_data_file = '/tmp/xtra3grc.bin'
+  assistance_url = 'http://xtrapath3.izatcloud.net/xtra3grc.bin'
```

Comment:
some things to consider here:
* file size sanity check before downloading. `/tmp/` is a tmpfs, so need to be careful about filling it up
* remove the file once it's uploaded to the module
* probably want some retry logic (e.g. going from wifi -> lte), and potentially put it in a thread to not block startup

Reply:
Can’t block or put in thread since this is only possible with the gps engine off. If it fails it fails, that’s ok.

will add file size check and delete